web/services/getTaskTypes.php

```diff
@@ -54,31 +55,15 @@
         break;
     }
 
-    // Hard-coded data
     $response['success'] = true;
```

Comment:
For now we can't remove it because this is how the front end knows it was successfully fetched: https://github.com/Igalia/phpreport/blob/main/web/js/tasks.js#L105